lib/backends/sdc/volumes.js

```diff
@@ -153,15 +159,78 @@ function listVolumes(params, options, callback) {
     assert.object(options, 'options');
     assert.func(callback, 'callback');
 
+    var filterName;
+    var filters;
+    var filterValue;
     var log = options.log;
+    var reqParams = {};
+    var valIdx;
+
+    /*
+     * By default, we only list volumes that are ready. As far as I know, docker
+     * doesn't have a concept of a volume's state, and so doesn't provide any
+     * ability to filter on such a property.
```

Comment:
Actually, I misread your question: with sdc-docker `docker volume create` returns only when the volume is in state `ready`, so at that point `docker volume ls` would include the volume that was just created in its output. Does that answer your question?